### PR TITLE
AK: Remove StringBuilder::build() in favor of to_deprecated_string()

### PR DESCRIPTION
**Pull request by linusg**

### CODE REVIEW

File: Userland/Libraries/LibC/syslog.cpp [hunk 1]
```diff
@@ -124,7 +124,7 @@ void vsyslog_r(int priority, struct syslog_data* data, char const* message, va_l
         combined.appendff("{}: ", get_syslog_ident(data));
 
     combined.appendvf(message, args);
-    DeprecatedString combined_string = combined.build();
+    DeprecatedString combined_string = combined.to_deprecated_string();
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibC/syslog.cpp:127]: Make this `auto`?

**Review by AtkinsSJ — COMMENTED** [review]
A bit pick

File: Userland/Libraries/LibC/syslog.cpp (hunk 1 above)
Comment (linusg) [reply Userland/Libraries/LibC/syslog.cpp]: I did a few of these but only looked for "builder.build()". Fixed!

**Comment by nico** [comment]
(doesn't build, though – overeager sed)

**Comment by linusg** [comment]
Yep, CLions "rename symbol" doesn't update nearly everything for some reason, and the subsequent manual find and replaces catches too much. Should be fixed now!

**Pull request merged**